### ISSUE TITLE
Projet 4 : Premiers tests

### ISSUE THREAD

**Issue by gabriel1628** [gabriel1628/Formation-Data-Scientist-OC#8]
- [x] Modéliser la target non normalisée
- [x] Comparer différents modèles
- [ ] Discrétiser les variables **YearBuilt** et **NumberofFloors**
- [x] Modéliser avec XGBoost sans encodage
- [ ] Modéliser la target normalisée par la superficie

Réfs : 
 - https://scikit-learn.org/stable/modules/preprocessing.html#preprocessing-discretization
 - fonctions `cut` et `qcut` de pandas
